react/Labs/PasswordInput/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import Icon from '../../Icon'
+import InputGroup from '../../InputGroup'
+import Input from '../../Input'
+import cx from 'classnames'
+import { getStrength } from './helpers'
+import styles from './styles.styl'
+import PropTypes from 'prop-types'
+
+const HideShowButton = props => {
+  const { hidden, className, ...rest } = props
+
+  return (
+    <button
+      type="button"
+      className={cx(styles.PasswordField__visibilityButton, className)}
+      {...rest}
+    >
+      <Icon
+        icon={hidden ? 'eye' : 'eye-closed'}
+        size={16}
+        color="var(--coolGrey)"
+      />
+    </button>
+  )
+}
+
+const Strength = props => {
+  const { password, className, ...rest } = props
+  const strength = getStrength(password)
+
+  return (
+    <progress
+      step="1"
+      min="0"
+      max="100"
+      value={strength.percentage}
+      className={cx(
+        styles.PasswordField__strength,
+        styles[`PasswordField__strength--${strength.label}`],
+        className
+      )}
+      {...rest}
+    />
+  )
+}
+
+const PasswordInput = props => {
+  const { className, showStrength, error, ...rest } = props
+  const [hidden, setHidden] = useState(true)
+
+  return (
+    <div className={cx(styles.PasswordField, className)}>
```

Comment:
Nit, but we should avoid mixing `styles.PasswordField` and `styles['PasswordField']` 😁